packages/components/src/inputs/src/TextArea.tsx

```diff
@@ -64,9 +64,9 @@ export interface TextAreaProps extends StyledComponentProps<RACTextFieldProps> {
      * This should only be used with the `showCharacterCount` prop.
      * If `true`, the TextArea prevents the text from ever going over the max length.
      * If `false`, the TextArea will allow the text to go over the max length, but it will add an error look tot he character count.
-     * @default true
+     * @default false
```

Comment:
I think in this case, we no longer need to set a default. The default isn't actually false, it's undefined.